### PR TITLE
Tweaks to hemtt

### PR DESCRIPTION
**Pull request by nk3nny**
- Version number 2.0.3
- Possible fix to fn_react to prevent players going prone

### When merged this pull request will:

1. *Describe what this pull request will do*
2. *Each change in a seperate line*

### CODE REVIEW

**Comment by nk3nny** [comment]
This is close to the current version 2.0.2 on steam.  I suggest it be discarded. Adding only the tweaks to fn_react. 

I see now that I must have been behind one step on the hemtt.tmol for some reason.  Leaving it here for reference. 

-k

File: .hemtt/base.toml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name = "lambs_danger"
+prefix = "lambs_danger"
```
Comment (diwako) [inline .hemtt/base.toml]: Wouldn't that change the prefix for everything, so addon main is then `lambs_danger_main` and the addon danger is then `lambs_danger_danger` ?

Comment (jokoho48) [reply .hemtt/base.toml]: yes I would not change that. also the "new" Template is as far I understood beeing reworked so i would not push that currently

**Comment by jokoho48** [comment]
i would Discard all changes expect the sqf file and use current master for continued dev

**Pull request closed without merging**